src/models/schema.sql

```diff
@@ -0,0 +1,45 @@
+-- create database livsiggenapi;
```

Comment:
```suggestion
create database livsiggenapi;
```

On peut commenter une pull request pour faire ce qu'on appelle des revues

Reply:
Non, c'est volontairement commenté car cette ligne ne doit pas s'exécuter à chaque fois.